modules/bg96/cellular_bg96.c

```diff
@@ -249,12 +249,14 @@ CellularError_t Cellular_ModuleEnableUE( CellularContext_t * pContext )
             cellularStatus = sendAtCommandWithRetryTimeout( pContext, &atReqGetNoResult );
         }
 
-        if( cellularStatus == CELLULAR_SUCCESS )
-        {
-            /* Enable RTS/CTS hardware flow control. */
-            atReqGetNoResult.pAtCmd = "AT+IFC=2,2";
-            cellularStatus = sendAtCommandWithRetryTimeout( pContext, &atReqGetNoResult );
-        }
+        #ifndef CELLULAR_BG96_DISABLE_FLOW_CONTROL
+            if( cellularStatus == CELLULAR_SUCCESS )
+            {
+                /* Enable RTS/CTS hardware flow control. */
+                atReqGetNoResult.pAtCmd = "AT+IFC=2,2";
```

Comment:
Is ```AT+IFC``` applicable to other modules as well?

Reply:
Good catch. I will update this PR to cover other modules.